entry_types/scrolled/package/src/frontend/useAtmo.js

```diff
@@ -21,12 +21,10 @@ export function AtmoProvider({children}){
       playerOptions: {tagName: 'audio', loop: true}
     });
     currentAtmo.multiPlayer = new MultiPlayer(currentAtmo.pool, {
-      loop: true,
       fadeDuration: 500,
       crossFade: true,
       playFromBeginning: false,
-      rewindOnChange: true,
-      pauseInBackground: true
```

Comment:
yes, it was a bug in master. I had discussed it with Tim. Did not wanted to create separate merge request and ticket for this change. Change is mentioned in the commit message.

Reply:
Sorry, I misinterpreted this change. You're right

Reply:
Maybe, in case you're reworking the commit message, you could provide the reason for this change there. Otherwise, for somebody who reads the commit history of this file, it could look like a case of 'first we did that, then we did that', but the reason would remain unclear.

Still, in cases like this, we often don't provide the reasoning. It would be an improvement to do so, but I don't consider it necessary for merge